docs/demo/scripts/customise_interactions.py

```diff
@@ -17,45 +17,67 @@
+# within WSIMOD is to store your customisations in separate modules which are
+# registered and applied when the model is initialised. Thus, to incrementally
+# build up the example we will store each customisation separately in its own module.
+# However, in practice one would contain these in a single module.
+#
+# But first...
+#
+# ## What are handlers?
 # %%
 from pprint import pprint as print
 
 from wsimod.nodes.sewer import Sewer
 
+
 my_sewer = Sewer(name="mr_sewer")
 print(my_sewer.push_set_handler)
 
 # %% [markdown]
 # We can see that `my_sewer` has two different functions (`push_set_land`
 # and `push_set_sewer`) stored in its `push_set_handler` dictionary. There are
-# four different entries in the handler (`'Demand'`, `'Land'`, `'Sewer'`,
-# `'default'`). This means that the `Sewer` node can respond to these four
+# four different entries in the handler (`Demand`, `Land`, `Sewer`,
+# `default`). This means that the `Sewer` node can respond to these four
 # different tags. Tags can take any value, but by convention they specify a
 # WSIMOD node type, or a tuple containing a node type and a specific type of
 # interaction (e.g., `('Demand','Garden')` in the
-# [land node](https://barneydobson.github.io/wsi/reference-land/#wsimod.nodes.land.Land)).
+# [land node](https://imperialcollegelondon.github.io/wsi/reference-land/#wsimod.nodes.land.Land)).
 # All nodes must have the dictionaries: `push_set_handler`, `push_check_handler`,
 # `pull_set_handler`, `pull_check_handler`, and at least one `default` key in each
-# (see the [Node](https://barneydobson.github.io/wsi/reference-nodes/#wsimod.nodes.nodes.Node)
+# (see the [Node](https://imperialcollegelondon.github.io/wsi/reference-nodes/#wsimod.nodes.nodes.Node)
 # class for defaults). If you do not want to define behaviour for a certain kind of interaction,
 # e.g., maybe you can never pull from this type of node, then you can use, e.g.,
-# the `pull_set_deny` and `pull_check_deny` functions for the `'default'` tag.
+# the `pull_set_deny` and `pull_check_deny` functions for the `default` tag.
 #
-# It is important to note that unless a tag is specified in the request, the `'default'`
+# It is important to note that unless a tag is specified in the request, the `default`
 # tag is used.
 # %% [markdown]
 # ## Illustration
-# We can illustrate the behaviour of handlers by creating a simple model.
-
+# We can illustrate the behaviour of handlers by creating a simple model. We will
+# create a temporary directory to save our model files in for the purpose of this
+# tutorial.
 # %%
 # Import packages
+import os
+import tempfile
+
 from wsimod.arcs.arcs import Arc
 from wsimod.core import constants
 from wsimod.nodes import Distribution, Node
+from wsimod.orchestration.model import Model, to_datetime
+
+# Identify the location of the scripts folder
+scripts_folder = os.path.join(os.path.abspath(""), "docs", "demo", "scripts")
```

Comment:
Yeah, this is not super elegant, but I know that jupyter and paths just don't get along very well...